pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -296,6 +307,31 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, log *
 	return nil
 }
 
+// needReImport returns true if and only if the source is a local reference and the target points to an api.ci image
+func needReImport(source, target *imagev1.ImageStream, tag string, targetImageStreamNotFound bool) bool {
+	if targetImageStreamNotFound {
```

Comment:
You don't need the targetImageStreamNotFound variable

Reply:
Updated.